.github/workflows/ci.yml

```diff
@@ -12,21 +12,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Fetch path to Zcash parameters
-        working-directory: ./zcash_proofs
-        shell: bash
-        run: echo "ZCASH_PARAMS=$(cargo run --release --example get-params-path --features directories)" >> $GITHUB_ENV
-      - name: Cache Zcash parameters
-        id: cache-params
-        uses: actions/cache@v3.3.2
-        with:
-          path: ${{ env.ZCASH_PARAMS }}
-          key: ${{ runner.os }}-params
-      - name: Fetch Zcash parameters
-        if: steps.cache-params.outputs.cache-hit != 'true'
-        working-directory: ./zcash_proofs
-        run: cargo run --release --example download-params --features download-params
```

Comment:
We can delete the `download-params` example now; it was only added for use in our CI. (However, we should verify that zebrad's CI isn't also using it; I think they might have been at one stage.)

Reply:
git grep shows no usage of download-params in the zebrad repo.